bom/runtime/pom.xml

```diff
@@ -182,7 +182,7 @@
         <mockito.version>3.3.3</mockito.version>
         <jna.version>5.3.1</jna.version>
         <antlr.version>4.7.2</antlr.version>
-        <quarkus-security.version>1.0.1.Final</quarkus-security.version>
+        <quarkus-security.version>1.1.0.Beta1</quarkus-security.version>
```

Comment:
Hi Stuart, it is not really related to this PR, would it make sense to move `quarkus-security` into `quarkus` eventually ? I've added a couple of exceptions into `quarkus-vertx-http` which should probably be in `quarkus-security`, and we can tweak things faster there

Reply:
by the way, I'm not asking to move it now :-), only a question/suggestion as I recall you were contemplating it at some point